apps/console/src/features/groups/components/edit-group/edit-group-roles-updated.tsx

```diff
@@ -0,0 +1,151 @@
+/**
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { Autocomplete, AutocompleteRenderGetTagProps, AutocompleteRenderInputParams } from "@mui/material";
+import TextField from "@oxygen-ui/react/TextField";
+import { IdentifiableComponentInterface, RolesMemberInterface } from "@wso2is/core/models";
+import { EmphasizedSegment, EmptyPlaceholder, Heading } from "@wso2is/react-components";
+import React, {
+    FunctionComponent,
+    HTMLAttributes,
+    ReactElement,
+    useEffect,
+    useState
+} from "react";
+import { useTranslation } from "react-i18next";
+import { getEmptyPlaceholderIllustrations } from "../../../core/configs/ui";
+import { GroupsInterface } from "../../models/groups";
+import { AutoCompleteRenderOption } from "../group-common-components/auto-complete-render-option";
+import { RenderChip } from "../group-common-components/render-chip";
+
+interface EditGroupRolesPropsInterface extends IdentifiableComponentInterface {
+    /**
+     * User profile
+     */
+    group: GroupsInterface;
+}
+
+export const EditGroupRoles: FunctionComponent<EditGroupRolesPropsInterface> = (
+    props: EditGroupRolesPropsInterface
+): ReactElement => {
+
+    const {
+        group
+    } = props;
```

Comment:
Let's keep this in a single line.

```suggestion
    const { group } = props;
```

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/4239/commits/7c5013e02a4dad911d69c42ac527b4ca9f2b7876